django/seasons/models.py

```diff
@@ -36,15 +36,21 @@ def __str__(self):
 #
 # class SeasonSubstitute(SeasonPlayer):
 #     positions = ChoiceArrayField(models.CharField(max_length=2, choices=POSITION_CHOICES))
-#
-#
-# class Team(models.Model):
-#     '''Team attributes that carry over season to season'''
-#     name = models.CharField(max_length=20, unique=True)
-#     # logo = models.ImageField(upload_to=logo_upload_to)
-#     # goal_horn = models.FileField(upload_to=goal_horn_upload_to)
-#
-#
+
+
+class Team(models.Model):
+    '''Team attributes that carry over season to season'''
+    name = models.CharField(max_length=20, unique=True)
+    # logo = models.ImageField(upload_to=logo_upload_to)
+    color_1 = models.CharField(max_length=30, help_text='CSS color value', blank=True)
+    color_2 = models.CharField(max_length=30, help_text='CSS color value', blank=True)
+    color_3 = models.CharField(max_length=30, help_text='CSS color value', blank=True)
+    # goal_horn = models.FileField(upload_to=goal_horn_upload_to)
```

Comment:
Maybe colors should be allowed to be changed season to season?

Reply:
Ability to change colors season-to-season seems a little out of scope for what we're trying to focus on.  It would be nice, but I wouldn't be surprised if teams entirely changed considering today's Slack Conversations.  I'd say don't worry about it right now

Reply:
agreed, thanks